### PR TITLE
crio: sync crio.conf

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: roles/docker/templates/crio.conf.j2 [hunk 1]
```diff
@@ -103,12 +103,17 @@ cgroup_manager = "systemd"
 # hooks_dir_path is the oci hooks directory for automatically executed hooks
 hooks_dir_path = "/usr/share/containers/oci/hooks.d"
 
+# default_mounts is the mounts list to be mounted for the container when created
+default_mounts = [
+	"/run/secrets",
```
Comment (runcom) [inline roles/docker/templates/crio.conf.j2]: This is wrong, should be:
```
/usr/share/rhel/secrets:/run/secrets
```

Comment (runcom) [reply roles/docker/templates/crio.conf.j2]: This will allow subscription manager to work inside a container

Comment (giuseppe) [reply roles/docker/templates/crio.conf.j2]: thanks, I've just updated it

File: roles/docker/templates/crio.conf.j2 [hunk 2]
```diff
@@ -103,12 +103,17 @@ cgroup_manager = "systemd"
...
+	"/usr/share/rhel/secrets:/run/secrets",
+]
+
 # pids_limit is the number of processes allowed in a container
 pids_limit = 1024
 
 # log_size_max is the max limit for the container log size in bytes.
 # Negative values indicate that no limit is imposed.
-log_size_max = 52428800
+log_size_max = -1
```
Comment (ashcrow) [inline roles/docker/templates/crio.conf.j2]: This was changed to ``52428800`` in our testing with @mrunalp. Should this go back to ``-1``?

Comment (giuseppe) [reply roles/docker/templates/crio.conf.j2]: not sure, I copied the value from the default configuration file.  In any case, if 52428800 worked, I am not going to change it :-)

Pushing a new version with the old value

**Comment by giuseppe** [comment]
just rebased

**Comment by ashcrow** [comment]
@sdodson the change and the failures here are unrelated. It's a configuration file fix.

**Pull request merged**